pkg/dataplane/deployment.go

```diff
@@ -313,7 +314,8 @@ func (d *Deployer) addCertMounts(
 				}
 				volumeName := GetServiceCertsSecretName(d.NodeSet, service.Name, certKey, 0)
 				if len(volumeName) > apimachineryvalidation.DNS1123LabelMaxLength {
-					volumeName = volumeName[:apimachineryvalidation.DNS1123LabelMaxLength]
+					hash := sha256.Sum224([]byte(volumeName))
+					volumeName = string(hash[:])
```

Comment:
doesn't produce a hex string, `hex.EncodeToString(hash[:])` from enconding/hex works

Reply:
since the hex string length is 56 maybe can add a short prefix e.g "cert", "cacert", "cm", "sec"